playground/src/PlaygroundView.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import * as tsdoc from '@microsoft/tsdoc';
+import './PlaygroundView.module.scss';
```

Comment:
```suggestion
import styles from './PlaygroundView.module.scss';
```

Reply:
I made these changes but for some reason the styling isnt applied if I set `className={styles.playgroundMainRow}`. Should I leave as is?

Reply:
I just left a note about needing to fix this later.